admin/customers.php

```diff
@@ -1396,17 +1414,17 @@ function init() {
                     $contents[] = array('text' => '<br>' . TEXT_INFO_GV_AMOUNT . ' ' . $currencies->format($customer_gv_balance));
 
                     $contents[] = array('text' => '<br>' . TEXT_INFO_NUMBER_OF_ORDERS . ' ' . $customers_orders->RecordCount());
-
                     if ($customers_orders->RecordCount() != 0) {
+
                       $lifetime_value = 0;
                       $last_order = array(
                         'date_purchased' => $customers_orders->fields['date_purchased'],
                         'order_total' => $customers_orders->fields['order_total'], 
                         'currency' => $customers_orders->fields['currency'], 
                         'currency_value' => $customers_orders->fields['currency_value'],
                       );
-                      foreach ($customers_orders as $result) {
-                        $lifetime_value += ($result['order_total'] * $result['currency_value']);
+                      foreach($customers_orders as $result) {
+                         $lifetime_value+= ($result['order_total'] * $result['currency_value']);
```

Comment:
seems like your editor added this.

Reply:
It might have, but what's the issue with those additional spaces?

Reply:
Argh, that must have been Beyond Compare "helping".